app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -1,11 +1,14 @@
 <%
+  local_assigns[:margin_bottom] ||= 6
```

Comment:
Are we sure we want to set this as the default?

Reply:
Yes, because 30px was what it was set to originally, and we don't want to change the default behaviour.